### PR TITLE
Rollups meta step

### PR DESCRIPTION
**Pull request by stephenctw**
Depends on https://github.com/cartesi/machine-solidity-step/pull/74 to pass the tests

### CODE REVIEW

**Comment by stephenctw** [comment]
An rollups integration test is added with a Lua orchestrator script which does the following:
- spawn an Anvil blockchain
- deploy smart contracts
- spawn an honest Rust Dave rollups node
- send some inputs
- spawn several Lua sybil nodes

As the code is working fine for now, I still think we need to:
- carefully review all the math/logic in the commitment calculation
- test more sybil combinations/edge cases (uarch-reset, send-cmio) at meta-step call
- polish the code with better engineering practice

To run the integration test with the following steps:
- run `make test-echo` from the `cartesi-rollups/tests/rollups` directory, this will print all Lua logs, including the sybil players and the orchestrator
- run `docker exec rollups-echo-test tail -f dave.log` on a separate terminal to see the logs produced by Rust Dave rollups node

File: cartesi-rollups/node/blockchain-reader/src/lib.rs [hunk 1]
```diff
@@ -29,18 +32,21 @@ use cartesi_dave_contracts::daveconsensus::DaveConsensus::EpochSealed;
 use cartesi_rollups_contracts::inputbox::InputBox::InputAdded;
 use rollups_state_manager::{Epoch, Input, InputId, StateManager};
 
+const DEVNET_CONCENSUS_ADDRESS: &str = "0x5FC8d32690cc91D4c39d9d3abcBD16989F875707";
```
Comment (guidanoli) [inline cartesi-rollups/node/blockchain-reader/src/lib.rs:35]: *consensus

Comment (stephenctw) [reply cartesi-rollups/node/blockchain-reader/src/lib.rs]: You are my hero

**Review by GCdePaula — DISMISSED** [review]
Great work!

**Review by GCdePaula — APPROVED** [review]
LGTM!
